inc/customizer/customizer.php

```diff
@@ -87,13 +87,16 @@ function css() {
 		// Start by importing Google web fonts
 		$return = '<style type="text/css" id="customizer-css">';
 
-		$import = array();
-		foreach ( $this->google_fonts as $font ) {
-			$import[ ] = urlencode( $font[ 0 ] ) . ':' . $font[ 1 ];
-		}
-		$import = array_unique( $import );
-		if ( !empty( $import ) ) {
-			$return .= '@import url(//fonts.googleapis.com/css?family=' . implode( '|', $import ) . '); ';
+		$import_font = apply_filters( 'vanage_import_google_fonts', true );
```

Comment:
Let's perhaps avoid adding a variable and using it once. How about:

`if ( $apply_filters( 'vanage_import_google_fonts', true ) ) {`